src/Canvas.tsx

```diff
@@ -182,12 +182,12 @@ function Canvas<R, K extends keyof R>({
 
   function getViewportRows() {
     const rowElements = [];
-    for (let idx = rowOverscanStartIdx; idx <= rowOverscanEndIdx; idx++) {
-      const row = rows[idx];
+    for (let rowIdx = rowOverscanStartIdx; rowIdx <= rowOverscanEndIdx; rowIdx++) {
+      const row = rows[rowIdx];
       rowElements.push(
         <RowRenderer<R, K>
-          key={idx}
-          idx={idx}
+          key={rowIdx}
```

Comment:
```suggestion
          key={row[rowKey]}
```
Shouldn't we do this instead? 🤔

Reply:
I remember thinking about it but do not remember why I did not change this. Let's revisit in the next PR

Reply:
We show also make `rowKey` a required prop before we make this change